vortex-file/src/open/mod.rs

```diff
@@ -139,11 +140,12 @@ impl VortexOpenOptions {
         read: R,
     ) -> VortexResult<VortexFile<FileIoDriver<R>>> {
         // Set up our segment cache.
-        let segment_cache = self
-            .segment_cache
-            .as_ref()
-            .cloned()
-            .unwrap_or_else(|| Arc::new(InMemorySegmentCache::default()));
+        let segment_cache = self.segment_cache.as_ref().cloned().unwrap_or_else(|| {
+            Arc::new(InMemorySegmentCache::new(
+                // For now, use a fixed 1GB overhead.
+                CacheBuilder::new(1 << 30),
```

Comment:
nit - I think we probably want to set the [eviction policy](https://docs.rs/moka/latest/moka/policy/struct.EvictionPolicy.html) to `lru`